test/tests/nuxeo-basics/run.sh

```diff
@@ -15,23 +15,33 @@ trap "docker rm -vf $cid > /dev/null" EXIT
 get() {
 	docker run --rm -i \
 		--link "$cname":nuxeo \
-		"$image" curl -fs \
-		-H "Content-Type:application/json" \
-		-u Administrator:Administrator \
-		http://nuxeo:8080/nuxeo/api/v1/$1 
+		--entrypoint curl \
+		"$image" \
+		-fs \
+			-H "Content-Type:application/json" \
+			-u Administrator:Administrator \
+			"http://nuxeo:8080/nuxeo/api/v1/$1"
 }
 
-. "./../../retry.sh" --tries "$NUXEO_TEST_TRIES" \
-	--sleep "$NUXEO_TEST_SLEEP" \
-	"get default-domain/workspaces"
+python3() {
+	docker run --rm -i \
+		--entrypoint python3 \
+		"$image" \
+		"$@"
+}
```

Comment:
We don't have `python` in the test environment, so we get to be a little creative. :smile: